setup.cfg

```diff
@@ -16,36 +16,36 @@ replace = __version__ = '{new_version}'
 
 [bumpversion:part:release]
 optional_value = final
-values =
+values = 
 	beta
 	final
 
 [bumpversion:file:docs/source/conf.py]
-search = version|release = {current_version}
+parse = version|release = {current_version}
 replace = {new_version}
 
 [bumpversion:file:Dockerfile]
-search = Version={current_version}
-replace = {new_version}
+search = Version="{current_version}"
+replace = Version="{new_version}"
```

Comment:
here also

Reply:
Bumpversion didn't work without those fixes. I think the behavior is different with parse and search.